packages/app-btc/src/operations/signTxn/index.ts

```diff
@@ -72,39 +68,48 @@ export const signTxn = async (
     meta: {
       version: signTxnDefaultParams.version,
       locktime: params.txn.locktime ?? signTxnDefaultParams.locktime,
-      inputSize: params.txn.inputs.length,
-      outputSize: params.txn.inputs.length,
-      hashType: params.txn.hashType ?? signTxnDefaultParams.hashtype,
+      inputCount: params.txn.inputs.length,
+      outputCount: params.txn.outputs.length,
+      sighash: params.txn.hashType ?? signTxnDefaultParams.hashtype,
     },
   });
+  const { metaAccepted } = await helper.waitForResult();
+  assertOrThrowInvalidResult(metaAccepted);
 
+  // duplicate locally and fill `prevTxn` if missing; we need completed inputs for preparing signed transaction
```

Comment:
Is this a TODO or a statement? because in this code we are not duplicating it locally

Reply:
no, not a todo. needs a change then

Reply:
changed. take a look